lib/configs/_base.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
         "consistent-return": "error",
         curly: "error",
         "default-case": "error",
+        "default-case-last": "error",
```

Comment:
@eslint-community/mysticatea-eslint-plugin Adding these as `'error'` was a breaking change, no?

We also support v6 with this plugin & people using v6 won't have these rules so ESLint will throw I think 🤔